mods/ra/rules/vehicles.yaml

```diff
@@ -24,14 +24,23 @@ V2RL:
 		Range: 4c0
 	Armament:
 		Weapon: SCUD
+	AmmoPool:
+		Ammo: 1
+		PipCount: 0
+		AmmoCondition: ammo
+	ReloadAmmoPool:
+		Delay: 240
+		Count: 1
 	AttackFrontal:
+	WithFacingSpriteBody@LOADED:
```

Comment:
There is already a `WithFacingSpriteBody:` that is coming from `^Vehicle`. Add a `-WithFacingSpriteBody:` above or remove `@Loaded:` so this'll replace the one coming from `^Vehicle` instead of adding a new one. First makes more sense to me as having @ part to describe what is really used for is better.

Reply:
You are fast! Okay, then I'll do the first one.

Reply:
Hmm, it seems not to work with both cases. The V2 is invisible and can't be selected.

Reply:
This still isn't adding a `-WithFacingSpriteBody:` nor removed the `@LOADED`. That means the game currently renders an additional loaded sprite all the time (although the empty sprite may be displayed on top of it so it isn't immediately noticable).

Reply:
Good catch.  Removing the `@LOADED` would be the better choice.